packages/sliderInput/components/SliderInput.tsx

```diff
@@ -0,0 +1,207 @@
+  /**
+   * Sets the current appearance of the input component. This defaults to InputAppearance.Standard, but supports `InputAppearance.Error` & `InputAppearance.Success` appearances as well.
+   */
+  appearance?: InputAppearance;
+  /**
+   * Whether or not the input is checked
+   */
+  checked?: boolean;
+  /**
+   * Unique identifier used for the input element
+   */
+  id: string;
+  /**
+   * The text or node content that appears next to the input
+   */
+  inputLabel: React.ReactNode | string;
+  /**
+   * Defaults to `true`, but can be set to `false` to visibly hide the content passed to `inputLabel`. The `inputLabel` should still be set even when hidden for accessibility support.
+   */
+  showInputLabel?: boolean;
+  /**
+   * The value being toggled
+   */
+  value?: string;
+  /**
+   * How the text content vertically aligns with the input
+   */
+  vertAlign?: "center" | "top";
+  /**
+   * hintContent is text or a ReactNode that is displayed directly under the input with additional information about the expected input.
+   */
+  hintContent?: React.ReactNode;
+  /**
+   * Sets the contents for validation errors. This will be displayed below the input element. Errors are only visible when the `TextInput` appearance is also set to `TextInputAppearance.Error`.
+   */
+  errors?: React.ReactNode[];
+}
+
+const SliderInput: React.FC<React.PropsWithRef<SliderInputProps>> = props => {
+  const {
+    appearance,
+    children,
+    disabled,
+    hintContent,
+    id,
+    inputLabel,
+    showInputLabel,
+    vertAlign,
+    checked,
+    value,
+    errors,
+    onBlur,
+    onFocus,
+    ...other
+  } = props;
+  const inputType = "Slider";
+  const inputDataCy = [
+    `${inputType}Input-input`,
+    ...(checked ? [`${inputType}Input-input.checked`] : []),
+    ...(appearance && appearance !== InputAppearance.Standard
+      ? [`${inputType}Input-input.${appearance}`]
+      : [])
+  ].join(" ");
+  const parentDataCy = [
+    `${inputType}Input`,
+    ...(checked ? [`${inputType}Input.checked`] : []),
+    ...(appearance && appearance !== InputAppearance.Standard
+      ? [`${inputType}Input.${appearance}`]
+      : [])
+  ].join(" ");
+  const [hasFocus, setHasFocus] = React.useState<boolean>(false);
+  const handleFocus = e => {
+    setHasFocus(true);
+
+    if (onFocus) {
+      onFocus(e);
+    }
+  };
+
+  const handleBlur = e => {
+    setHasFocus(false);
+
+    if (onBlur) {
+      onBlur(e);
+    }
+  };
+
+  return (
+    <FormFieldWrapper id={id} errors={errors} hintContent={hintContent}>
+      {({ getValidationErrors, isValid, describedByIds, getHintContent }) => (
+        <div className={vAlignChildren} data-cy={parentDataCy}>
+          <label
+            className={cx(
+              flex({
+                align: vertAlign === "top" ? "flex-start" : "center"
+              }),
+              display("inline-flex"),
+              bruteForceKillLabelMargin // to override CNVS bottom margin on <label>
```

Comment:
we still use CNVS? 😒

Reply:
I'm actually not sure. this was very much copy 🍝  from `ToggleComponent`

Reply:
I removed this and didn't notice any changes, so I left it out for now.